### PR TITLE
Step1 jose

### PR DESCRIPTION
**Pull request by josephum**
New StockDetails entity and some test data in data.sql

### CODE REVIEW

File: src/main/resources/data.sql [hunk 1]
```diff
@@ -14,6 +14,11 @@ VALUES (1, '2022-04-15 00:00:00', FALSE, 1, '2022-05-07 00:00:00', 'computer', T
        (2, '2022-03-17 00:00:00', FALSE, 2, '2022-05-20 00:00:00', 'sports', TRUE, 7, 'Shoe', 1, 'ACTIVE', 7, 2, 'Pieces'),
        (3, '2022-02-05 00:00:00', FALSE, 3, '2022-04-12 00:00:00', 'TV', TRUE, 43, 'TV', 1, 'ACTIVE', 43, 5, 'Pieces');
 
+INSERT INTO stock_details (i_date, quantity, price, remaining_quantity, product_id)
+VALUES ('2022-04-15 00:00:00', 2344, 999.99, 34525423, 3),
+       ('2022-01-15 00:00:00', 67, 99.99, 9687, 1),
```
Comment (MehmetKaldirim) [inline src/main/resources/data.sql:19]: I wanna buy a few from your computers my friend, they are just 99.99 dollars ...:)

**Review by MehmetKaldirim — APPROVED** [review]
fetched and controlled ...Work properly...Good Job...

File: src/main/java/com/example/accountingapp/entity/StockDetails.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.time.LocalDateTime;
+
+@Entity
+@Data
+@NoArgsConstructor
```
Comment (VitalySlabada) [inline src/main/java/com/example/accountingapp/entity/StockDetails.java:13]: curious why you did not include is deleted.
I wonder if I should also remove it from my table:) 
I approve but maybe we can discuss when we meet

**Review by VitalySlabada — APPROVED** [review]
tested, looks good!

File: src/main/java/com/example/accountingapp/entity/StockDetails.java (hunk 2 above)
Comment (josephum) [reply src/main/java/com/example/accountingapp/entity/StockDetails.java:13]: Good catch!
This is one of the few entities without is_deleted in the Database Design (E-R Diagram)

File: src/main/resources/data.sql (hunk 1 above)
Comment (josephum) [reply src/main/resources/data.sql:19]: Man, I dream of :)))

**Review by MehmetKaldirim — APPROVED** [review]

**Review by AbbosB12 — APPROVED** [review]

**Pull request merged**